src/modules/modem_module.c

```diff
@@ -612,29 +613,52 @@ static void on_all_states(struct modem_msg_data *msg)
 	}
 }
 
-static void message_handler(struct modem_msg_data *msg)
+static void modem_module_thread_fn(void)
```

Comment:
The _fn suffix is not included in the other modules for the thread function. I think it could stay and the other modules could change. Also, we could remove the `modem` prefix. Just call it ```module_thread_fn``` in all thread dependent modules.

Reply:
Agreed, removed the prefix
Other modules can be updated in separate PR